tasks.py

```diff
@@ -0,0 +1,154 @@
+"""
```

Comment:
Though many of these functions are quite intuitive, I think it'd be worthwhile to at least have some function level comments on them. You don't need to put parameter level comments, though I do prefer them, but I think we should have at least Class, function, etc. comments in place.

Reply:
Good suggestion. I added some doc-strings and task help which would show up when using `invoke --help deploy` for example